proxy/src/main/java/org/apache/accumulo/proxy/ProxyServer.java

```diff
@@ -1766,13 +1765,9 @@ public void importDirectory(ByteBuffer login, String tableName, String importDir
       org.apache.accumulo.proxy.thrift.AccumuloException,
       org.apache.accumulo.proxy.thrift.AccumuloSecurityException, TException {
     try {
-      ImportMappingOptions loader = getConnector(login).tableOperations().importDirectory(importDir)
-          .to(tableName);
-      if (setTime) {
-        loader.tableTime().load();
-      } else {
-        loader.load();
-      }
+      getConnector(login).tableOperations().importDirectory(importDir).to(tableName)
```

Comment:
I have to agree. Normally, I'd be against boolean params, but in the case of single-param fluent APIs, I think there's a good exception to be made.

In some cases, it could still be problematic, as in:

```java
// does false mean feature isn't explicitly enabled, or that it is explicitly disabled?
var myObject = myBuilder.newInstance().enableFeature(false).build();
```

I don't think that applies here, though. I think it's clear enough here.

Reply:
It had previous only been explicitly enabled and always false otherwise.  This makes logic internally simpler but the case where the user has a boolean to determine the behavior more cumbersome.  Now the boolean param is explicit "setValue" with the default staying the same as false.